### PR TITLE
Created PR-template

### PR DESCRIPTION
**Pull request by anbclausen**
Making our PR's more consistent.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# This solves issue
+#[ISSUE NUMBER]
+
+# Comments
+Please provide any useful comments about the PR.
+
+# I have
+- Been assigned to this issue
+- Used TDD
+- Commited after each TDD rhythm
```
Comment (soer7022) [inline .github/pull_request_template.md:10]: Make this into a checklist

**Review by soer7022 — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (anbclausen) [reply .github/pull_request_template.md]: Fixed.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# This solves issue
+#[ISSUE NUMBER]
```
Comment (soer7022) [inline .github/pull_request_template.md:2]: If we write `closes #ISSUE NUMBER` it will automatically close the issue when the PR is merged

Comment (soer7022) [reply .github/pull_request_template.md:2]: At least I think so

Comment (dangerstone) [reply .github/pull_request_template.md:2]: But what if the issue is only partially solved? If we close the issue after merging, then we can't PR and get feedback before it's completely finished.

Comment (soer7022) [reply .github/pull_request_template.md:2]: Good point, maybe just having a reference to the issue number is fine

Comment (anbclausen) [reply .github/pull_request_template.md:2]: Resolved? :+1:

**Review by soer7022 — APPROVED** [review]

**Review by dangerstone — APPROVED** [review]

**Pull request merged**